extensions/wikia/AchievementsII/Ach_setup.php

```diff
@@ -227,30 +227,28 @@ function Ach_UserPreferences( $user, &$preferences ) {
 /*
  * Used in the mantainance/wikia/takeAchievementsRankingSnapshot.php script
  */
-function Ach_TakeRankingSnapshot($force = false) {
+function Ach_TakeRankingSnapshot( $force = false ) {
 	global $wgCityId;
 	$dbw = WikiFactory::db( DB_MASTER );
 
-	$res = $dbw->select('ach_ranking_snapshots', array('date'), array('wiki_id' => $wgCityId), __METHOD__);
+	$res = $dbw->select( 'ach_ranking_snapshots', array( 'date' ), array( 'wiki_id' => $wgCityId ), __METHOD__ );
 	$rankingService = new AchRankingService();
 
-	if($row = $dbw->fetchObject($res)) {
-		if(strtotime($row->date) <= (time() - (60*60*24)) || $force) {
-			$dbw->update('ach_ranking_snapshots', array('date' => date('Y-m-d H:i:s'), 'data' => $rankingService->serialize()), array('wiki_id' => $wgCityId), __METHOD__);
-			echo("\t* Snapshot for the wiki with ID {$wgCityId} has been updated\n");
-			$dbw->commit(__METHOD__);
-		}
-			else {
-			echo("\t* A user ranking snapshot already exists for the wiki with ID {$wgCityId} and is still valid (taken on {$row->date})\n");
+	if( $row = $dbw->fetchObject( $res ) ) {
+		if( strtotime( $row->date ) <= ( time() - ( 60 * 60 * 24 ) ) || $force ) {
+			$dbw->update( 'ach_ranking_snapshots', array( 'date' => date( 'Y-m-d H:i:s' ), 'data' => $rankingService->serialize() ), array( 'wiki_id' => $wgCityId ), __METHOD__ );
+			echo( "\t* Snapshot for the wiki with ID {$wgCityId} has been updated\n" );
+			$dbw->commit( __METHOD__ );
+		} else {
+			echo( "\t* A user ranking snapshot already exists for the wiki with ID {$wgCityId} and is still valid (taken on {$row->date})\n" );
 		}
-	}
-	else {
-		$dbw->insert('ach_ranking_snapshots', array('wiki_id' => $wgCityId, 'date' => date('Y-m-d H:m:s'), 'data' => $rankingService->serialize()), __METHOD__);
-		echo("\t* Snapshot for the wiki with ID {$wgCityId} has been taken\n");
-		$dbw->commit(__METHOD__);
+	} else {
+		$dbw->insert( 'ach_ranking_snapshots', array( 'wiki_id' => $wgCityId, 'date' => date( 'Y-m-d H:m:s' ), 'data' => $rankingService->serialize() ), __METHOD__ );
+		echo( "\t* Snapshot for the wiki with ID {$wgCityId} has been taken\n" );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the parentheses from this "echo" call. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2041)